src/additional-access-entries.tf

```
@@ -0,0 +1,8 @@
+locals {
+  # If you have custom access entries and policies, create a file called `additional-access-entries_override.tf`
+  # and in that file override any of the following declarations as needed.
+  #
+  # For example, we use this to add a policy for the GitHub Actions OIDC role
+  overridable_access_entry  = []
+  overridable_access_policy = []
```

```suggestion
  overridable_access_map    = {}
  overridable_access_entry  = []
  overridable_access_policy = []
```

Then change https://github.com/cloudposse-terraform-components/aws-eks-cluster/blob/8e6f6b716177b7145bda9a745e39c524d20b2214/src/main.tf#L53

to

```hcl
  access_entry_map = merge(
    local.aws_team_roles_access_entry_map,
    local.aws_sso_access_entry_map,
    local.iam_roles_access_entry_map,
    local.iam_users_access_entry_map,
    local.overridable_access_map)
```

---

added